cmd/gvisor/gvisor.go

```diff
@@ -17,13 +17,16 @@ limitations under the License.
 package main
 
 import (
+	"flag"
 	"log"
 	"os"
 
 	"k8s.io/minikube/pkg/gvisor"
 )
 
 func main() {
+	flag.Parse()
```

Comment:
Is this needed?

Reply:
Not any longer. It was when I accidentally started using glog =)